app/app/services/provider_search_service.rb

```diff
@@ -27,11 +34,17 @@ def search(query = "")
       # Use Pinwheel's results if there aren't any results from Argyle, or
       # there is an exact match from Pinwheel.
       results = pinwheel_results if results.length == 0 || any_exact_matches?(pinwheel_results, query)
+
+      results = filter_results(results, BLOCKED_PINWHEEL_EMPLOYERS)
     end
 
     results
   end
 
+  def filter_results(results, blocked_employers)
+    results.select { |result| !result.provider_options.provider_id.in? blocked_employers }
```

Comment:
Here's a slightly more readable alternative:

```suggestion
    results.select { |result| blocked_employers.exclude?(result.provider_options.provider_id) }
```